plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -3217,6 +3217,22 @@ public void testProjectionPushdownOnPartitionedTables()
         }
     }
 
+    @Test
+    public void testProjectionPushdownOnPartitionedTablesWithComments()
+    {
+        try {
+            assertUpdate("CREATE TABLE table_with_partition_comments (id BIGINT, qid BIGINT COMMENT 'QID', root ROW(f1 BIGINT, f2 BIGINT)) WITH (partitioning = ARRAY['id'])");
```

Comment:
Can you also put a comment on the `root` column, either here or in a separate test?